src/App.js

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
 import './App.css';
 import { Home, About, Contact, Navigation } from './components';
+import { Route } from 'react-router-dom';
 
 const App = () => (
   <div>
+    <Home />
+    <About />
     <Navigation />
+    <Contact />
```

Comment:
Great job rendering your components, but refactor to use Routes! We want to render these components using <Route /> components so we can set up their URL pathing.